warehouse/core/src/main/java/datawave/mr/bulk/BulkInputFormat.java

```diff
@@ -54,8 +56,6 @@
 import org.apache.accumulo.core.security.TablePermission;
 import org.apache.accumulo.core.singletons.SingletonReservation;
 import org.apache.accumulo.core.util.Pair;
```

Comment:
Pair can be replaced with [AbstractMap.SimpleEntry](https://docs.oracle.com/javase/8/docs/api/java/util/AbstractMap.SimpleEntry.html) at a later date.